appveyor.yml

```diff
@@ -0,0 +1,11 @@
+environment:
+  matrix:
+    - TOXENV: py36
```

Comment:
Please use "…-coverage".
Can we have py38 maybe instead already?

Reply:
Will do. I was figuring that only the Linux builds needed coverage. Do you want the full tox suite py27,34,..,38 and pypy? Or is just running a 38 sufficient?

Reply:
Hmm... looks like the way you have coverage set up won't play nicely with AppVeyor. Do we need to have coverage for windows builds?

Reply:
I've just thought it would be good to have the Windows code being covered, too.
A single build should suffice.

What's the problem with AV?

Reply:
I see:
```
py38-coverage run-test: commands[1] | 'C:\projects\pdbpp/.ci/report-coverage.sh'
ERROR: InvocationError for command 'C:\projects\pdbpp/.ci/report-coverage.sh' (exited with code 8)
```

(via https://ci.appveyor.com/project/blueyed/pdbpp/builds/28897748)

Not much information there.
Would be great if you could test / fix it from trying it locally.

Reply:
You can already add `set -x` there maybe, which is good in general, and might show what fails.

Reply:
In short, your tox env is trying to run a shell script (`coverage-report.sh`) in a Windows (CMD/PowerShell) environment.

Now, AppVeyor does support running bash scripts ([source "Run Coverage Analysis on AppVeyor"](https://www.appveyor.com/blog/2017/03/17/codecov/)):
```
after_test:
  - ps: |
      $env:PATH = 'C:\msys64\usr\bin;' + $env:PATH
      Invoke-WebRequest -Uri 'https://codecov.io/bash' -OutFile codecov.sh
      bash codecov.sh -f "coverage.xml" -t YourUploadToken
```

but I don't know how to integrate that with tox.

Reply:
Tox should take care of that (via https://github.com/pdbpp/pdbpp/blob/1f6571ad8dcb0d1bb5e4987996f6c4a3d1676a9d/tox.ini#L19).
Maybe it needs other slashes?

Or we just need the above env (PATH) adjustment for AppVeyor in general?

Reply:
@dougthor42 
Any more insights here?  Have you tried it locally in Windows?
I've pushed some commits to investigate / work on that, since I think we really should have coverage with it.